run.py

```diff
@@ -174,6 +174,17 @@ def main(args):
   from nab.detectors.skyline.skyline_detector import SkylineDetector
   from nab.detectors.random.random_detector import RandomDetector
   from nab.detectors.null.null_detector import NullDetector
+  detectors = []
+  for detector in args.detectors:
+      if ':' in detector:
+        package, name = detector.split(':', 2)
+        className = detectorNameToClass(name)
```

Comment:
I wouldn't include this step; it's probably better (more straightforward) to have users specify the class name upfront, rather than understand what the `detectorNameToClass()` method does. So your original example would be `run.py -d myproject.detectors.awesome_detector:AwesomeDetector`.